ui/app/styles/core/breadcrumb.scss

```diff
@@ -11,7 +11,24 @@
     }
   }
 
+  ul {
+    align-items: center;
+  }
+
+  li::before {
+    font-size: x-large;
```

Comment:
This changes causes a small move in the file explorer breadcrumb. We could limit this to `.breadcrumb.is-large` elements

![ezgif com-gif-maker(1)](https://user-images.githubusercontent.com/775380/146461965-6925a99f-922a-4fc0-a142-a568d40dbe43.gif)